values.yaml

```diff
@@ -380,7 +389,19 @@ st2web:
   securityContext: {}  # NB: nginx requires some capabilities, drop ALL will cause issues.
   # mount extra volumes on the st2web pod(s) (primarily useful for k8s-provisioned secrets)
   ## Note that Helm templating is supported in 'mount' and 'volume'
-  extra_volumes: []
+  extra_volumes:
```

Comment:
extra_volumes is only for user-provided volumes so that if someone sets it to an empty list it doesn't break chart functionality.